README.md

```diff
@@ -114,10 +131,14 @@ Note that in the following instructions, we assume your catkin workspace (in whi
    ```
 
 1. Install avoidance module dependencies (pointcloud library and octomap).
-
-   ```bash
-   sudo apt install libpcl1 ros-kinetic-octomap-* ros-kinetic-yaml-*
-   ```
+   - Melodic
+     ```bash
+     sudo apt install libpcl1 ros-melodic-octomap-* ros-melodic-yaml-*
+     ```
+   - Kinetic
+     ```bash
+     sudo apt install libpcl1 ros-kinetic-octomap-* ros-kinetic-yaml-*
```

Comment:
The last dependency fails (`ros-kinetic-yaml-*`). How do I get it/is it needed?

```
ubuntu@ubuntu:~/github$ sudo apt install ros-melodic-yaml-*
Reading package lists... Done
Building dependency tree       
Reading state information... Done
E: Unable to locate package ros-melodic-yaml-*
E: Couldn't find any package by glob 'ros-melodic-yaml-*'
E: Couldn't find any package by regex 'ros-melodic-yaml-*'
```

Reply:
@mrivi ^^^^

Reply:
I don't have it on my system so I think it isn't required on 18.04

Reply:
Thanks. Removed.